sections/combat.tex

```diff
@@ -209,6 +209,8 @@ \subsection*{Campaign and Cooperative Combat}
   \includegraphics[width=0.97\linewidth]{\art/dread_knight.jpg}
 \end{center}
 
+\clearpage
+
```

Comment:
Is it necessary?

Reply:
It is. If there is a shorter text, the title from next page starts to creep in. It happened to me several times - this prevents that.

<img width="599" alt="Screenshot 2024-06-12 at 18 52 26" src="https://github.com/Heegu-sama/Homm3BG/assets/1928291/fe9d62a0-639c-4b4c-b811-32f7eb965343">